src/features/lightspeed/utils/explorerView.ts

```diff
@@ -15,7 +15,11 @@ export function getWebviewContentWithLoginForm(
     "explorer",
     "main.js",
   ]);
-  const styleUri = getUri(webview, extensionUri, ["media", "style.css"]);
+  const styleUri = getUri(webview, extensionUri, [
+    "media",
+    "lightspeedExplorerView",
```

Comment:
:+1: 

I think `.vscodeignore` file also needs to be updated to include `media/lightspeedExplorerView/style.css` in the .vsix package.